ts/features/payments/bizEventsTransaction/components/PaymentsBizEventsTransactionCartList.tsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import { View } from "react-native";
+import Placeholder from "rn-placeholder";
+import {
+  HSpacer,
+  IOStyles,
+  ListItemTransaction,
+  VSpacer
+} from "@pagopa/io-app-design-system";
+import { CartItem } from "../../../../../definitions/pagopa/biz-events/CartItem";
+import { formatAmountText } from "../utils";
+
+type Props = {
+  carts?: ReadonlyArray<CartItem>;
+  loading: boolean;
+  onPress: (cartItem: CartItem) => void;
+};
+
+/**
+ * This component renders a list of transaction cart details
+ */
+export const PaymentsBizEventsTransactionCartList = ({
+  carts,
+  loading,
+  onPress
+}: Props) => {
+  if (loading) {
+    return <SkeletonTransactionDetailsList />;
+  }
+  if (!carts) {
+    return <></>;
+  }
+
+  return (
+    <>
+      {carts.map((cartItem, index) => (
+        <ListItemTransaction
+          key={index.toString()}
```

Comment:
We should never use index as component keys. This could lead to problems, like items being replaced inside lists.
Instead, try to use something unique (like an ID).

Reply:
We don't have an ID for these items but only a `refNumberValue` that I'm not pretty sure is unique. Anyway I've putted it but if the backend says that it is not an unique value I think that putting the index is the only way (even because the max number of elements that can be added is not more than 3).

Addressed into 90bdac3da7e0ddc57949cbdd54da09aa5dc0d231